rex/exploit/technique.py

```diff
@@ -258,6 +258,20 @@ def _read_in_global_data_with_gets(self, data):
         data_bvv  = self.crash.state.solver.BVV(data)
         return read_to, (glob_data == data_bvv)
 
+    def _find_str_address(self, s):
```

Comment:
you know clemory.find exists, right?

Reply:
I actually don't. I'll rewrite this part. thanks